src/sst/core/statapi/statengine.cc

```diff
@@ -86,12 +86,12 @@ StatisticProcessingEngine::~StatisticProcessingEngine()
     }
 }
 
-bool StatisticProcessingEngine::registerStatisticCore(StatisticBase* stat, uint8_t comp_load_level)
+bool StatisticProcessingEngine::registerStatisticCore(StatisticBase* stat)
 {
     if ( stat->isNullStatistic() )
         return true;
 
-    uint8_t stat_load_level = comp_load_level == STATISTICLOADLEVELUNINITIALIZED ? m_statLoadLevel : comp_load_level;
+    uint8_t stat_load_level = m_statLoadLevel;
```

Comment:
We can have a component load level, which needs to be preserved. I think we ended up changing this logic.